go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	cosmossdk.io/api v0.3.1
```

Comment:
I am not sure why this was changed, but it seems like go changes the file consistently for me.